acceptance/tests/load_libfacter.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'Ruby can load libfacter'
+
+agents.each do |agent|
+  if agent.platform.variant == 'windows'
+    facter_gem = 'C:/Program Files/Puppet Labs/Puppet/facter/lib/facter.rb'
+    path = '/cygdrive/c/Windows/system32:/cygdrive/c/Windows'
+  else
+    facter_gem = '/opt/puppetlabs/puppet/lib/ruby/vendor_ruby/facter.rb'
+    path = '/usr/local/bin:/bin:/usr/bin'
+  end
+
+  on agent, "env PATH='#{path}:#{agent['privatebindir']}' ruby -rubygems '#{facter_gem}'" do
```

Comment:
👍 - great to see this make it into an acceptance test.

One minor difference in the test I used was passing `-I` like:

```
"C:\Program Files\Puppet Labs\Puppet\sys\ruby\bin\ruby.exe" -I"C:\Program Files\Puppet Labs\Puppet\facter\lib" -rubygems "C:\Program Files\Puppet Labs\Puppet\facter\lib\facter.rb"
```

I think your test is probably OK given `-I` clearly doesn't make a difference if this test is passing, but wanted to point that out.

```
-I directory
used to tell Ruby where to load the library scripts. Directory path will be added to the load-path variable (`$:').
```